<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modificamezzo_window</class>
 <widget class="QMainWindow" name="modificamezzo_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modifica Mezzo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xz.qrc">
    <normaloff>:/images/images/AnconAmbiente_Logo.png</normaloff>:/images/images/AnconAmbiente_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="modificamezzo_widget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>120</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Fine iscrizione Albo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Targa:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annulla_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>ANNULLA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targa_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="patente_combo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>220</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Patente Non Necessaria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="modello_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Modello:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Allestimento:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="iscrizione_datepicker">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>220</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="salva_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>SALVA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="allestimento_field">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>185</y>
      <width>120</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Patente Necessaria</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>225</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Stato:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stato_combo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>220</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Regolare</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manutenzione</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="targa_error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>235</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="modello_error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>235</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="allestimento_error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>235</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xz.qrc"/>
 </resources>
 <connections/>
</ui>
